server/webhook.go

```diff
@@ -18,56 +18,73 @@ import (
 	"github.com/mattermost/mattermost-server/model"
 )
 
-type JIRAWebhookIssue struct {
-	Self   string
-	Key    string
-	Fields struct {
-		Assignee    *jira.User
-		Reporter    *jira.User
-		Summary     string
-		Description string
-		Priority    *struct {
-			Id      string
-			Name    string
-			IconURL string
-		}
-		IssueType struct {
-			Name    string
-			IconURL string
-		}
-		Resolution *struct {
-			Id string
-		}
-		Status struct {
-			Id string
-		}
-		Labels []string
-	}
+const (
+	eventIssueCreated = uint64(1 << iota)
+	eventIssueDeleted
+	eventUnresolvedIssueDeleted
+	eventIssueAssigned
+	eventIssueReopened
+	eventIssueResolved
+	eventIssueUpdatedStatus
+	eventIssueUpdatePriority
+	eventIssueUpdateSummary
+	eventIssueUpdateDescription
+	eventIssueUpdateSprint
+	eventIssueUpdateRank
+	eventIssueUpdatedAttachment
+	eventIssueUpdatedLabels
+	eventCommentCreated
+	eventCommentDeleted
+	eventCommentUpdated
+	eventMax = iota
+)
+
+const maskLegacy = eventIssueCreated |
```

Comment:
see https://github.com/mattermost/mattermost-plugin-jira/blob/c6f5fc6df455522509fbe95e22d94a9ee620b3d3/server/webhook.go#L63-L79 and https://github.com/mattermost/mattermost-plugin-jira/blob/c6f5fc6df455522509fbe95e22d94a9ee620b3d3/server/webhook.go#L146-L151